main.go

```diff
@@ -196,6 +157,7 @@ func main() {
 	fmt.Println()
 	log.Printf("System installed cocoapods version:")
 
+	// pod can be in the PATH as an rbenv shim and pod --version will return "rbenv: pod: command not found"
```

Comment:
should not we install cocoapods before calling version?

Reply:
yes we should.